src/RedisCache/RedisCache/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 
+## Version 1.3.0-preview
```

Comment:
please put release note under upcoming release and remove line 22

Reply:
Updated